src/components/Select.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react'
+
+import { SelectOption } from './SelectOption'
+import { Select, SelectProps } from './Select'
+import componentDecorators from '../../.storybook/component-decorators'
+
+export default {
+  title: 'Components/Select',
+  component: Select,
+  parameters: {
+    a11y: {
+      /*
+    Disabling "nested-interactive" rule until there's a fix from Ionic: https://github.com/ionic-team/ionic-framework/issues/23511
+     */
```

Comment:
I have to imagine you wrote this on one line and prettier or something did this disgusting format?
```suggestion
      // Disabling "nested-interactive" rule until there's a fix from Ionic: https://github.com/ionic-team/ionic-framework/issues/23511
```

Reply:
Ya exactly. I’ll adjust on Monday.